spyderlib/widgets/tests/test_array_builder.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2009- The Spyder Development Team
+# Licensed under the terms of the MIT License
+#
+
+"""
+Tests for the Numpy Matrix/Array Builder Widget.
+"""
+
+# Third party imports
+from qtpy.QtCore import Qt
+from pytestqt import qtbot
+import pytest
+
+# Local imports
+from spyderlib.widgets.arraybuilder import NumpyArrayDialog
+
+
+# --- Fixtures
+# -----------------------------------------------------------------------------
+@pytest.fixture
+def botinline(qtbot):
+    dialog = NumpyArrayDialog(inline=True)
+    qtbot.addWidget(dialog)
+    qtbot.dialog = dialog
```

Comment:
You don't need this if you return `dialog` directly (same for the other fixtures)